packages/plans-grid/src/plans-table/plan-item.tsx

```diff
@@ -48,6 +49,8 @@ export interface Props {
 	onToggleExpandAll?: () => void;
 	allPlansExpanded: boolean;
 	disabledLabel?: string;
+	CTAVariation: 'FULL_WIDTH' | 'NORMAL';
+	popularBadgeVariation: 'ON_TOP' | 'NEXT_TO_NAME';
```

Comment:
We should export these values from the package similar to how domain picker has done [here](https://github.com/Automattic/wp-calypso/blob/master/packages/domain-picker/src/domain-picker/suggestion-item.tsx#L22-L23) so we can use it like [this](https://github.com/Automattic/wp-calypso/pull/47318/files#diff-a88807ae898bfacf7689a16c723644ee8d5ddf40677fd0ce119761a404d0b897R10)